tests/ragger/boilerplate_client/utils.py

```diff
@@ -0,0 +1,75 @@
+from io import BytesIO
```

Comment:
This functions don't seem to be boilerplate specific, it would be nice to have a common utils package

Reply:
To be cleaned in another PR